test.sh

```diff
@@ -29,9 +29,15 @@ function module_exists {
   [ $(fs_cli_command module_exists "$module") = 'true' ]
 }
 
+function http_check {
+  local url="$1"
+  local code="$2"
```

Comment:
Personally, I like to do:
```bash
local url="$1"; shift
local code="$1"; shift
```
...because then if you provide too few arguments it fails.